tests/components/harmony/test_switch.py

```diff
@@ -35,78 +19,17 @@ async def test_connection_state_changes(
     await hass.config_entries.async_setup(entry.entry_id)
     await hass.async_block_till_done()
 
-    # mocks start with current activity == Watch TV
-    assert hass.states.is_state(ENTITY_WATCH_TV, STATE_ON)
```

Comment:
I didn't find a way to keep switches tests with disabled entities by default :(

Reply:
Mock the entity registry, register the switches, and then enable them so they appear an are testable.

There is a `mock_registry` that you can use from `tests.common`